src/components/CheckoutForm.js

```diff
@@ -77,29 +88,24 @@ class CheckoutForm extends Component {
         </Form>
       );
     }
-
-    if (this.state.success !== "") {
-      paymentStatus = (
-        <Card>
-          <Card.Content header={this.state.success} />
-        </Card>
-      );
-    }
-
-    if (this.state.error !== "") {
-      paymentStatus = (
-        <Card>
-          <Card.Content header={this.state.error} />
-        </Card>
-      );
-    }
     return (
       <Container>
-        {paymentStatus}
         {stripeForm}
       </Container>
     );
   }
 }
 
-export default injectStripe(CheckoutForm);
+const mapDispatchToProps = {
```

Comment:
Can you please explain how/why this variable is used?

Reply:
We're subscribing to the redux state on this component. The display flash messages is set up in redux, and we have to call it in so we can use them

Reply:
Got it.